src/DIRAC/Resources/Storage/GFAL2_StorageBase.py

```diff
@@ -14,6 +14,8 @@
 * DIRAC_GFAL_GRIDFTP_SESSION_REUSE: This should be exported
   and set to true in server bashrc files for efficiency reasons.
 
+* DIRAC_GFAL_GRIDFTP_ENABLE_IPV6: this should be eported and set
```

Comment:
```suggestion
* DIRAC_GFAL_GRIDFTP_ENABLE_IPV6: this should be exported and set
```

Reply:
For alphabetic order, this variable should come first

Reply:
done